Scripts/Proxies/DefaultProxyRay.cs

```diff
@@ -139,24 +91,66 @@ public void SetLength(float length)
 
 		public void SetColor(Color c)
 		{
+			m_LineRenderer.SetColors(c, c);
```

Comment:
hm... removing this fails to set the color properly. I don't have time to look into why right now, but I'll figure it out.